source/gameengine/GameLogic/SCA_VibrationActuator.h

```diff
@@ -0,0 +1,59 @@
+/*
```

Comment:
The GPL license block should be without the "The Original Code is Copyright (C) 2001-2002 by NaN Holding BV.". To avoid this frequent error I made a wiki page : https://github.com/UPBGE/blender/wiki/GPL-license-block.